src/components/RecipeCard.tsx

```diff
@@ -2,11 +2,13 @@ import React from 'react'
 import { View, Image, StyleSheet, Text } from 'react-native'
 import { RecipeCardProps } from "../../types"
 
-export default function RecipeCard({ id, rcpImage}: RecipeCardProps) {
+export default function RecipeCard({ id, rcp}: RecipeCardProps) {
 
+        //TODO: Display the titles of the recipes
         return (
             <View>
-                <Image key={id} source={{uri: rcpImage}} style={styles.picture} />
+                {/* <Text>{rcp.title}</Text> */}
+                <Image key={id} source={{uri: rcp.image}} style={styles.picture} />
```

Comment:
I get another warning saying URI should not be an empty string. I'm assuming when we get to the end of our map function, rcp.image is undefined. Replacing rcp.image with 'rcp.image || " "' would fix the warning and give source a string that will result in the same behavior.